code/graphics/opengl/gropengl.cpp

```diff
@@ -1020,6 +1021,9 @@ int opengl_init_display_device()
 	attrs.height = (uint32_t) gr_screen.max_h;
 
 	attrs.title = Osreg_title;
+	if (Window_title != "") {
```

Comment:
This should be `if (!Window_title.empty())`.

Reply:
Fair enough.